lib/src/main/java/com/cloudbees/groovy/cps/CpsTransformer.java

```diff
@@ -1050,25 +1081,23 @@ public void visitDeclarationExpression(final DeclarationExpression exp) {
             makeNode("sequence", new Runnable() {
                 @Override
                 public void run() {
+                    // TODO: Possibly move this logic into multipleAssignment(...) or replace that with more specific
+                    // logic.
                     for (Expression e : exp.getTupleExpression().getExpressions()) {
                         final VariableExpression v = (VariableExpression) e;
                         makeNode("declareVariable", new Runnable() {
                             @Override
                             public void run() {
-                                loc(exp);
```

Comment:
Why?

Reply:
Because! More specifically, because we want to invoke https://github.com/cloudbees/groovy-cps/blob/master/lib/src/main/java/com/cloudbees/groovy/cps/Builder.java#L211 - the current code would actually never have worked in the first place, since there's only the above and https://github.com/cloudbees/groovy-cps/blob/master/lib/src/main/java/com/cloudbees/groovy/cps/Builder.java#L215. I also don't think the location is particularly relevant for our use here, since we're just gonna assign it shortly afterwards.